cmd/prom-migrator/main.go

```diff
@@ -154,7 +167,8 @@ func parseFlags(conf *config, args []string) {
 		"set this to the remote write storage that the migrator is writing along with the progress-enabled.")
 	flag.BoolVar(&conf.progressEnabled, "progress-enabled", true, "This flag tells the migrator, whether or not to use the progress mechanism. It is helpful if you want to "+
 		"carry out migration with the same time-range. If this is enabled, the migrator will resume the migration from the last time, where it was stopped/interrupted. "+
-		"If you do not want any extra metric(s) while migration, you can set this to false. But, setting this to false will disble progress-metric and hence, the ability to resume migration.")
+		"If you do not want any extra metric(s) while migration, you can set this to false. But, setting this to false will disable progress-metric and hence, the ability to resume migration.")
+
 	// Authentication.
 	// TODO: Auth/password via password_file and bearer_token via bearer_token_file.
```

Comment:
Can this be removed with this PR?

Reply:
Do you mean the todo for accepting auth via files?